readme.md

```diff
@@ -23,6 +23,24 @@ var newObject = mapObj({foo: 'bar'}, function (key, value, object) {
 //=> {bar: 'foo'}
 ```
 
+## API
+
+### mapObj(source, mapFn, [target]);
+ 
+- `source` - the source object to copy properties from.
+
+- `mapFn` - the mapping function.
+  
+  It has signature `mapFn(sourceKey, sourceValue, source)`. 
+  
+  It must return a two item array: `[targetKey, targetValue]`.
+  
+- `target` - *optional* - specify the target object to map properties on to. 
+
+  An empty object will be created if not supplied.
+ 
+**Returns:** `target`
+
```

Comment:
@jamestalmage Hah, you pretty much found the only module that doesn't use my normal style. Probably because it's better documented by the MDN docs, since it's a ponyfill. But no worries, I can easily do the changes myself ;)